### ISSUE TITLE
cannt find module /lib/index.js

### ISSUE THREAD

**Issue by koala819** [jquense/docusaurus-theme-tailwind#2]
I clone,
I make a yarn install
Go to www to run yarn start
```diff
-Cannot find module 'docusaurus-theme-tailwind/lib/index.js'

-Please verify that the package.json has a valid main entry
```

**Comment by diegoaguilar** [comment]
@koala819 did you figure this out?

**Comment by koala819** [comment]
Yes with this issue found in Docusaurus : [link](https://github.com/facebook/docusaurus/issues/2961)

**Comment by armandopbringas** [comment]
> I clone, I make a yarn install Go to www to run yarn start
> 
> ```diff
> -Cannot find module 'docusaurus-theme-tailwind/lib/index.js'
> 
> -Please verify that the package.json has a valid main entry
> ```

@koala819 i had the same problem but i´d never cloned anything, i just setup a docusaurus app from their respective docs and i tried to follow this issue, so which repo you are meaning that you cloned?
